src/test/resources/application-test.yml

```diff
@@ -27,23 +25,28 @@ application:
       initialDelay: 0
       lockAtLeastFor: PT1S
 
+aws:
+  provider: localstack
+
```

Comment:
I'm not sure that this is still needed - don't think it's used anywhere.

Reply:
removed